runner/operator/access/v1_0_0/structural_variants/__init__.py

```diff
@@ -0,0 +1,126 @@
+        Create all sample inputs for all runs triggered in this instance of the operator
+
+        :return: list of json_objects
+        """
+        # Get all completed runs for the given request ID
+        request_id_runs = Run.objects.filter(
+            tags__requestId=self.request_id,
+            status=RunStatus.COMPLETED
+        )
+
+        # Get all standard bam ports for these runs
+        standard_bam_ports = Port.objects.filter(
+            name='standard_bams',
+            run__id__in=[r.id for r in request_id_runs]
+        )
+
+        # Filter to only tumor bam files
+        # Todo: Use separate metadata fields for Tumor / sample ID designation instead of file name
+        standard_tumor_bam_files = [f for p in standard_bam_ports for f in p.value if TUMOR_OR_NORMAL_SEARCH in f['path'].split('/')[-1]]
+        sample_ids = [f['path'].split('/')[-1].split(SAMPLE_ID_SEP)[0] for f in standard_tumor_bam_files]
+
+        sample_inputs = []
+        for i, b in enumerate(standard_tumor_bam_files):
+            sample_input = self.construct_sample_inputs(
+                sample_ids[i],
+                b
+            )
+            sample_inputs.append(sample_input)
+
+        return sample_inputs
+
+    def get_jobs(self):
+        """
+        Convert job inputs into serialized jobs
+
+        :return: list[(serialized job info, Job)]
+        """
+        sample_inputs = self.get_sample_inputs()
+
+        return [
+            (
+                APIRunCreateSerializer(
+                    data={
+                        'name': "ACCESS LEGACY SV M1: %s, %i of %i" % (self.request_id, i + 1, len(sample_inputs)),
+                        'app': self.get_pipeline_id(),
+                        'inputs': job,
+                        'tags': {
+                            'requestId': self.request_id,
+                            'cmoSampleIds': job["sv_sample_id"]
+                        }
+                    }
+                ),
+                job
+             )
+            for i, job in enumerate(sample_inputs)
+        ]
+
+    def construct_sample_inputs(self, tumor_sample_id, tumor_bam):
+        """
+        Use sample metadata and json template to create inputs for the CWL run
+
+        :return: JSON format sample inputs
+        """
+        with open(os.path.join(WORKDIR, 'input_template.json.jinja2')) as file:
+            template = Template(file.read())
+
+            tumor_sample_names = [tumor_sample_id]
+            tumor_bams = [{
+                "class": "File",
+                "location": 'juno://' + tumor_bam['path']
+            }]
+
+            normal_bam = FileRepository.filter(
+                file_type='bam',
+                path_regex=ACCESS_DEFAULT_SV_NORMAL_FILENAME
+            )
+
+            if not len(normal_bam) == 1:
+                msg = "Incorrect number of files ({}) found for ACCESS SV Default Normal".format(len(normal_bam))
+                raise Exception(msg)
```

Comment:
Did you want to capture this exception in the logs? I think you can just add

```
logger.exception(msg)
```

before you `raise` it